### ISSUE TITLE
Interger Scale is fixed at a height of 240, should be 224.

### ISSUE THREAD

**Issue by cheatfreak47** [libretro/picodrive#26]
Sega Genesis/Mega Drive outputs at 320 x 224, but when enabling Integer Scale and setting aspect to 1:1 PAR you see distortions caused by it being fixed at 240.

Should hopefully a simple fix. (:

I first noticed this on the 3DS builds but it seems to be a problem for all of them.

**Comment by hizzlekizzle** [comment]
As discussed in IRC, it looks like some games actually need 240 px:
https://github.com/libretro/picodrive/blob/master/platform/libretro/libretro.c#L450-L458

For reference, genplusgx pads the 224 out to 240 with 8 px border on top and bottom and then has core options for disabling those borders.

The other option would be to use AV_SET_GEOMETRY or whatever to drop down to the proper resolution after load.

**Comment by AkimanBengus** [comment]
I have written a fix for this issue a few weeks ago, I will soon commit my work and you will be able to try it.
In fact some games change the resolution in real time from 256x224 to 320x224 (ex: sf2ce).
And some games use a resolution of 320x240 in PAL mode and 320x224 in NTSC (ex: street of rage)

My fix will take care of these real time changes.

I also added a "core option" to choose which between PAR and DAR.

**Comment by hizzlekizzle** [comment]
Oh man, sounds great :D

**Comment by AkimanBengus** [comment]
I commit my work on my Fork: https://github.com/AkimanBengus/picodrive
Can you download and compile it to try the fix with your 3DS before I ask for a Pull request.
This commit can normally fix these issues #7 and #15.
To test it you can download the 240p rom and set the region to europe in core option.
After restarting the rom you will see the last option: PAL VDP 320x**240**p and when you push A button the system will change to PAL VDP 320x**224**p.
Normally the picture will not stretch when the resolution change.

**Comment by fr500** [comment]
@AkimanBengus the best way to get people to try stuff is merging to master, worst case scenario we can force push later, building for 3DS is not trivial for most people

I'm thinking about a way to have another branch in our repos to stage pull requests but it would take a while to implement.

TL;DR: send a PR :+1:

**Comment by AkimanBengus** [comment]
Ok, I will do a Pull Request :-)

**Comment by orbea** [comment]
@cheatfreak47 Is this still an issue?

**Comment by Antiriad** [comment]
Hmmm it seems that I have the exact opposite issue? O_o 
Issue #49

**Comment by cheatfreak47** [comment]
Oh right, yeah I think this was fixed.
